inc/cleantalk-updater.php

```diff
@@ -1314,5 +1314,10 @@ function apbct_update_to_6_54_1()
         $apbct->settings['data__email_decoder_encode_email_addresses'] = 0;
     }
 
+    if ( ! isset($apbct->settings['forms__gravityforms_save_spam']) ) {
```

Comment:
Probably this is redundant. We have runAutoSaveStateVars() here:
https://github.com/CleanTalk/wordpress-antispam/blob/54274f6e353e10ca4e1ab55ad4a740a2dcedf1b0/lib/Cleantalk/ApbctWP/State.php#L414 during updater work

Reply:
If the option is exists, no need to change it to default value. The case is very synthetic, but possible.